modules/configuring-hybrid-ovnkubernetes.adoc

```diff
@@ -40,7 +56,7 @@ After creating the file, several network configuration files are in the `manifes
 +
 [source,terminal]
 ----
-$ ls <installation_directory>/manifests/cluster-network-*
+$ ls -1 <installation_directory>/manifests/cluster-network-*
```

Comment:
For consistency, this is formatted the same as [Modifying advanced network configuration parameters
](https://docs.openshift.com/container-platform/4.6/installing/installing_azure/installing-azure-network-customizations.html#modifying-nwoperator-config-startup_installing-azure-network-customizations). The `-1` is kind of a personal preference, isn't it? Or is there more of a need for this?

Reply:
The only reason for the `-1` is because then it matches the output.